tokens/eth/0x38c6A68304cdEfb9BEc48BbFaABA5C5B47818bb2.json

```diff
@@ -1,34 +1,37 @@
 {
   "symbol": "HPB",
-  "name": "HPBCoin",
+  "name": "High Performance Blockchain",
   "type": "ERC20",
   "address": "0x38c6A68304cdEfb9BEc48BbFaABA5C5B47818bb2",
   "ens_address": "",
   "decimals": 18,
-  "website": "",
+  "website": "https://hpb.io",
   "logo": {
     "src": "",
     "width": "",
     "height": "",
     "ipfs_hash": ""
   },
   "support": {
-    "email": "",
+    "email": "hpb@hpb.io",
     "url": ""
   },
   "social": {
-    "blog": "",
+    "blog": "https://hpb.io/blog",
     "chat": "",
     "facebook": "",
     "forum": "",
-    "github": "",
+    "github": "https://github.com/hpb-project/",
     "gitter": "",
     "instagram": "",
     "linkedin": "",
     "reddit": "",
     "slack": "",
-    "telegram": "",
+    "telegram": "https://t.me/hpbglobal",
     "twitter": "",
     "youtube": ""
-  }
-}
\ No newline at end of file
+  },
+  "deprecation": {
+    "migration_type": "newchain:auto:269",
+    "time": "2018-10-14T18:01:00Z"
+}
```

Comment:
there seems to be a closing bracket missing - CI complains about invalid json

Reply:
you're right, this is now fixed.